src/library/importer.py

```diff
@@ -1,17 +1,20 @@
 from django_tenants.utils import schema_context
+from library.utils import get_library_schema_name
 from quest_manager.admin import QuestResource
 from quest_manager.models import Quest
 
-from .utils import library_schema_context
 
-
-def import_quests_to(*, destination_schema, quest_import_ids):
+def import_quests_to(*, destination_schema, quest_import_ids, source_schema=None):
```

Comment:
Can you add a function description to the docstring?  What does this do and where is it used?